worker/src/RTC/IceCandidate.cpp

```diff
@@ -1,39 +1,66 @@
 #define MS_CLASS "RTC::IceCandidate"
 // #define MS_LOG_DEV_LEVEL 3
 
-#include "RTC/IceCandidate.hpp"
 #include "Logger.hpp"
+#include "RTC/IceCandidate.hpp"
 
 namespace RTC
 {
-	/* Instance methods. */
+	/* Class methods. */
 
-	flatbuffers::Offset<FBS::WebRtcTransport::IceCandidate> IceCandidate::FillBuffer(
-	  flatbuffers::FlatBufferBuilder& builder) const
+	IceCandidate::CandidateType IceCandidate::CandidateTypeFromFbs(
+	  FBS::WebRtcTransport::IceCandidateType type)
 	{
-		MS_TRACE();
```

Comment:
Please don't remove `MS_TRACE()` calls.

Reply:
I didn't. `IceCandidate::FillBuffer` has it's `MS_TRACE`. The diff just shows it moved.